qiskit/ignis/characterization/coherence/fitters.py

```diff
@@ -101,6 +102,33 @@ def __init__(self, backend_result, xdata,
                                      circuit_names, expected_state='0',
                                      time_index=1, time_unit=time_unit)
 
+    def guess_params(self, qind=0):
```

Comment:
Why is the T2* fitter being touched, it is not a BaseCalibrationFitter?

Reply:
Just doing the calibrations (the T2* is used to calibrate frequency) I realized it needed better support for guessing the fit parameters